lib/elastomer/client/delete_by_query.rb

```diff
@@ -0,0 +1,82 @@
+module Elastomer
+  class Client
+
+    class DeleteByQuery
+
+      def initialize( client, query, params = nil )
```

Comment:
Definitely a separate PR for that sweep. And the guilty party should be the one writing that PR as penance.